containers/Config.ts

```diff
@@ -0,0 +1,54 @@
+import { ethers } from "ethers";
+type Network = ethers.providers.Network;
+export const config = (network: Network | null) => {
```

Comment:
I like passing in `network` here because you could imagine all of the config changing with a diff network

Reply:
ye, I was thinking we could re-use this file for addresses even for network switching (I know there is `EMPs.ts` but this could join all configs in one place for ease of access.